docs/user-manual/en/amqp-server-connections.md

```diff
@@ -0,0 +1,230 @@
+
+For a `receiver`, the broker will create a message producer on an address, that will *receive* messages from another AMQP endpoint.
+
+Both elements will work like a message bridge however without any extra weight required to process messages. They are just like any other consumer or producer as part of ActiveMQ Artemis Broker.
+
+Senders and Receivers can be specified by address matching using [wildcard expressions](wildcard-syntax.md).
+
+There are two properties you can use to configure a sender or receiver:
+
+- match: for the wildcard expression mathing addresses
+- queue-name: The name of a queue, without expressions
+
+Example of configuring senders and receivers using address expressions:
+
+```xml
+<broker-connections>
+ <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+    <sender match="queues.#"/>
+    <!-- notice the local queues for removequeues.# need to be created on this broker -->
+    <sender match="remotequeues.#"/>
+ </amqp-connection>
+</broker-connections>
+
+<addresses>
+ <address name="remotequeues.A">
+    <anycast>
+       <queue name="remoteQueueA"/>
+    </anycast>
+ </address>
+ <address name="queues.B">
+    <anycast>
+       <queue name="localQueueB"/>
+    </anycast>
+ </address>
+</addresses>
+```
+
+Example of configurig senders and receivers using queue names:
+
+```xml
+<broker-connections>
+    <amqp-connection uri="tcp://MY_HOST:MY_PORT" name="my-broker">
+        <sender queue-name="remoteQueueA"/>
+        <sender queue-name="localQueueB"/>
+    </amqp-connection>
+</broker-connections>
+
+<addresses>
+     <address name="remotequeues.A">
+        <anycast>
+           <queue name="remoteQueueA"/>
+        </anycast>
+     </address>
+     <address name="queues.B">
+        <anycast>
+           <queue name="localQueueB"/>
+        </anycast>
+     </address>
+</addresses>
+
+```
+*Important:* Notice that on the `receiver` case we will only match local existent queues, so if you are using receivers make sure you pre-create the queue locally otherwise we broker will not be able to match remote queues and addresses.
+
+*Important:* Be careful to not create a sender and a receiver to the same destination, otherwise they will incur in infinite echoes of sends and receives.
+
+
+# Peers
+A peer broker connection element will be a mix of sender and receivers. The ActiveMQ Artemis broker will create both a sender and a receiver for a peer element, and the endpoint should know how to deal with the pair without creating an infinite feedback of sending and receiving messages.
+
+Currently [Apache QPID Dispatch Router](https://qpid.apache.org/components/dispatch-router/index.html) as a peer. ActiveMQ Artemis will create the pair of receivers and sender for each matching destination, and these senders and receivers will have special configuration to let qpid dispatch know to collaborate with an ActiveMQ Artemis Broker.
```

Comment:
Apache QPID Dispatch Router (i.e not QPID). Then the sentence is broken and needs completed (e.g 'can act as a peer'.  Then later Qpid Dispatch (rather than qpid dispatch). It might be good if "waypoint" was mentioned since thats what its doing, setting the broker up as a waypoint.